README.md

```diff
@@ -15,7 +15,7 @@ The following services are available via HTTP API, C++ library interface and Nod
 
 To quickly try OSRM use our [demo server](http://map.project-osrm.org) which comes with both the backend and a frontend on top.
 
-For a quick introduction about how the road network is represented in OpenStreetMap and how to map specific road network features have a look at [this guide about mapping for navigation](https://www.mapbox.com/mapping/mapping-for-navigation/).
+For a quick introduction about how the road network is represented in OpenStreetMap and how to map specific road network features have a look at [this guide about mapping for navigation](https://web.archive.org/web/20221206013651/https://labs.mapbox.com/mapping/mapping-for-navigation/).
```

Comment:
May be worth putting link to some "official" OSM wiki page here? E.g. https://wiki.openstreetmap.org/wiki/Routing ?

Reply:
+1

Reply:
I think the old Mapbox guide is pretty useful as well still. I added a link to the wiki as well. What do you think?

Reply:
LGTM :+1: